packages/controller-utils/src/perfTestChecksum.ts

```diff
@@ -0,0 +1,216 @@
+// import { toChecksumHexAddress } from './util';
```

Comment:
Do we want to remove that file entirely now? Having perf test files seems like a good idea though 😅 but that can come later

Reply:
It's definitely not production grade perf test, but something like `reassure` could be useful.

Reply:
Then I guess, we can just remove it for now? But I'll let the wallet framework team to decide 😄

Reply:
Yea removed, it was commited by accident.